jattoabdul/narwhalorders
# feat(home): Add css styling to home page

**What does this PR do?**
- [x] It adds styling to the home page to make it look nicer
- [x] it adds responsiveness to the page

**Description of Task to be completed?**
>- After the app is installed, the app should monitor for new orders. When a new order is received it should check to see if there are any products in the order that contains the word “Narwhal”. If there are, then it should add a record in the DB or somewhere. We’ll call these “Narwhal Orders”.
>- When a merchant accesses the app through their admin, the app should show a responsive list of Narwhal Orders.

**How should this be manually tested?**
- Go to the app URL,
- Install the app
- Create a new order or Pay for an order
- Load the app in your Shopify admin and see the newly-designed home page.

**Any background context you want to provide?**
- Link to Figma designs can be accessed [here](https://www.figma.com/proto/ooqOb8G1YTK4trDTrA9srb/Narwhal-App?node-id=1%3A2&scaling=min-zoom)

**What are the relevant issues?**
- [#4 ](https://github.com/jattoabdul/narwhalorders/issues/4)

**Screenshots (if appropriate)**
- 
<img width="1434" alt="Screenshot 2020-06-10 at 1 02 05 AM" src="https://user-images.githubusercontent.com/12856872/84212563-16b5bb80-aab6-11ea-9210-c906c63c0002.png">

**Questions:**
- N/A
